tools/clang/lib/CodeGen/CodeGenTypes.cpp

```diff
@@ -47,7 +47,7 @@ CodeGenTypes::~CodeGenTypes() {
 
   for (llvm::FoldingSet<CGFunctionInfo>::iterator
        I = FunctionInfos.begin(), E = FunctionInfos.end(); I != E; )
-    delete &*I++;
+    ::operator delete(&*I++);
```

Comment:
This prevents calling the destructor, which is maybe working because the destructors are trivial, but we probably shouldn't rely on that.